extensions/HuggingFace/python/src/aiconfig_extension_hugging_face/remote_inference_client/conversational.py

```diff
@@ -256,7 +250,7 @@ async def run_inference(
 
         response = client.translation(**completion_data)
```

Comment:
Don't we also need to update this to `client.conversational`?

Reply:
updated, mistakenly kept it while testing.